src/layouts/app/layout/side-menu.tsx

```diff
@@ -63,21 +94,101 @@ const SideMenu: React.FC<SideMenuProps> = ({ menu }) => {
   const { open } = useControllerData(controller);
   const { updateCtrl } = useControllerActions(controller);
   const handleDrawerClose = useCallback(() => updateCtrl({ open: false }), [updateCtrl]);
+
+  const freeHeight = useFreeHeight()
+  + 64; // mt
+
   return <StyledDrawer variant="permanent" open={open}>
-    <Stack direction="row" justifyContent="flex-end" alignItems="center" sx={{ height: HEADER_HEIGHT }}>
-      <IconButton onClick={handleDrawerClose} sx={{ mr: 1 }}>
-        <ChevronLeftIcon />
-      </IconButton>
+
+    <Stack direction="column" justifyContent="space-between" alignItems="center" sx={{ height: freeHeight + 64 }}>
+      <Box width="100%">
+        <Stack direction="row" justifyContent="flex-end" alignItems="center" sx={{ height: HEADER_HEIGHT }}>
+          <IconButton onClick={handleDrawerClose} sx={{ mr: 1 }}>
+            <ChevronLeftIcon />
+          </IconButton>
+        </Stack>
+        <Divider />
+        <List>
+          { menu.map((item, index) => <ItemByType key={index} {...item} />) }
+        </List>
+      </Box>
+      { config.DEBUG && <Languages name="Language" icon={Language} list={languages} isActive={() => false}/> }
     </Stack>
-    <Divider />
-    <List>
-      { menu.map((item, index) => <ItemByType key={index} {...item} />) }
-    </List>
   </StyledDrawer>;
 };
 
 export default SideMenu;
 
+export interface LanguagesProps {
+  name: string
+  icon?: SvgIconComponent
+  list: Array<LanguageItem>
+  isActive: (location: Location) => boolean
```

Comment:
are you trying to check active language?  why do you need location? Why can't it be inside component?